index.html

```diff
@@ -10,43 +10,66 @@
 <link rel="stylesheet" href="styles.css">
 <body>
     <header class="header">
-        <a href="#" class="nabar-brand"> My logo</a>
-    <nav class="navbar">        
-        <i class="fa fa-bars" class="menu-icon"></i>        
+        <a href="#" class="navbar-brand"> My logo</a>
+    <nav class="navbar">
+        <div class="hamburger">        
+            <i class="fa fa-bars" class="menu-icon"></i>
+        </div>
```

Comment:
very good use of HTML semantics, however, please try t use better class names like header-container, this will help you in doing your css, because you know exactly which class you are targeting